environment-dev.yml

```diff
@@ -41,18 +46,16 @@ dependencies:
   - fastparquet
   # for release script
   - pygit2
+  # TODO(https://github.com/modin-project/modin/issues/5194): Uncap xgboost
+  # when we use collective instead of rabit.
+  - xgboost>=1.7.1,<2.0.0
+  - tqdm
   - pip:
-      # TODO(https://github.com/modin-project/modin/issues/5194): Uncap xgboost
-      # when we use collective instead of rabit.
-      - xgboost>=1.7.1,<2.0.0
       # Fixes breaking ipywidgets changes, but didn't release yet.
       - git+https://github.com/modin-project/modin-spreadsheet.git@49ffd89f683f54c311867d602c55443fb11bf2a5
-      - tqdm
       - git+https://github.com/airspeed-velocity/asv.git@ef016e233cb9a0b19d517135104f49e0a3c380e9
-      - ray[default]>=1.4.0,<2.1.0
+      # no conda package for windows
```

Comment:
what does this comment do?

Reply:
I tried to install this package using conda, however this package is only available for linux and macOS. https://anaconda.org/conda-forge/connectorx/files

Reply:
https://github.com/sfu-db/connector-x/issues/413